admin.py

```diff
@@ -800,17 +800,41 @@ def BuildSensorList(self, software_version):
         recording |= 1
       if sensor.supports_min_max_recording:
         recording |= 2
+      try:
+          range_min=int(sensor.range_min)
+      except:
+          range_min = None
+      try:
+          range_max=int(sensor.range_max)
+      except:
+          range_max = None
+      try:
+          normal_min=int(sensor.normal_min)
+      except:
+          normal_min = None
+      try:
+          normal_max=int(sensor.normal_max)
+      except:
+          normal_max = None 
+      try:
+          unit=str(sensor.unit)
```

Comment:
And then perhaps another one for str, although I'd expect this may only fail if the string is None.